app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  has_many :posts
```

Comment:
Please add the foreign key attribute set to author_id to ensure that the creator of each post is saved as author not user as described in the database.